indico/modules/events/editing/client/js/components/EditingView/Footer.jsx

```diff
@@ -0,0 +1,78 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2020 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+import logoURL from 'indico-url:assets.image';
+import configURL from 'indico-url:core.config';
+
+import React, {useState} from 'react';
+import {Image, Modal} from 'semantic-ui-react';
+import {useIndicoAxios} from 'indico/react/hooks';
+import {Translate} from 'indico/react/i18n';
+
+import './Footer.module.scss';
+
+export default function Footer() {
+  const [tosModal, setTosModal] = useState(false);
+
+  const {data} = useIndicoAxios({
+    url: configURL(),
+    camelize: true,
+    trigger: 'once',
+  });
+
+  const config = data;
+  if (!config) {
+    return null;
+  }
+
+  const {helpURL, hasTos, contactEmail, tosHtml} = config;
+  return (
+    <div styleName="editing-footer">
+      <div styleName="logo">
+        <Image src={logoURL({filename: 'indico_small.png'})} styleName="image" />
```

Comment:
Like this there's currently no way to override the footer logo like we do everywhere else (e.g. to put the CERN logo in the footer), but I don't see any good solution here for now, so I think we need to live with this for now, and find some solution for it later that does not involve react overrides / plugins (we can't expect people to write a plugin for this, and we don't want to do this for our own customizations since it's moer maintenance).